### PR TITLE
Removed code related to bff

### PR DESCRIPTION
**Pull request by acn-dgopa**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.AccessManagement/appsettings.json [hunk 1]
```diff
@@ -9,12 +9,11 @@
     "RightsCacheTimeout": 5
```
Comment (acn-dgopa) [inline src/Altinn.AccessManagement/appsettings.json:9]: will revert changes to appsetttings files

File: src/Altinn.AccessManagement.Core/Configuration/GeneralSettings.cs [hunk 2]
```diff
@@ -14,10 +14,5 @@ public class GeneralSettings
         /// Option to disable csrf check
         /// </summary>
         public bool DisableCsrfCheck { get; set; }
```
Comment (jonkjetiloye) [inline src/Altinn.AccessManagement.Core/Configuration/GeneralSettings.cs:16]: Isn't the CSRF check also moved?

**Review by jonkjetiloye — COMMENTED** [review]
Other stuff that can/should be removed as BFF resumes responsibility:
- ProfileController
- LookUpController.GetOrganisation

But these can't be removed before BFF is updated to not call AM but the register/profile APIs directly and can be done in a separate issue if you prefer

**Review by jonkjetiloye — APPROVED** [review]

**Comment by acn-dgopa** [comment]
> Other stuff that can/should be removed as BFF resumes responsibility:
> 
> * ProfileController
> * LookUpController.GetOrganisation
> 
> But these can't be removed before BFF is updated to not call AM but the register/profile APIs directly and can be done in a separate issue if you prefer

Yes, Have to sync both the repositories for that change. I will create a separate issue for the same

**Pull request merged**